src/Widgets/FooterWidget/FooterWidgetEditingConfig.ts

```diff
@@ -0,0 +1,9 @@
+import { provideEditingConfig } from 'scrivito'
+import { FooterWidget } from './FooterWidgetClass'
+import Thumbnail from './thumbnail.svg'
+
+provideEditingConfig(FooterWidget, {
+  description: 'Shows the footer of the homepage.',
+  thumbnail: Thumbnail,
+  title: 'Footer',
```

Comment:
* [ ] FYI Maybe "Homepage Footer" would make the purpose clearer and we could drop the description.